### ISSUE TITLE
Multiline types create false positives with the indent rule

### ISSUE THREAD

**Issue by kaicataldo** [eslint/typescript-eslint-parser#430]
**What version of TypeScript are you using?**
2.6.2

**What version of `typescript-eslint-parser` are you using?**
11.0.0

**What code were you trying to parse?**

```ts
const foo: {
  bar: string
} = {};
```

**What did you expect to happen?**
No errors

**What happened?**
```sh
  2:1  error  Expected indentation of 4 spaces but found 2  indent
  3:1  error  Expected indentation of 4 spaces but found 0  indent

✖ 2 problems (2 errors, 0 warnings)
  2 errors, 0 warnings potentially fixable with the `--fix` option.
```

I'll try to find some time to work on this.

**Comment by JamesHenry** [comment]
Thanks @kaicataldo 👍

**Comment by JamesHenry** [comment]
I'm going to close this due to lack of activity.

We now have a brand new way of running integration tests for this project, which allows for complex setups to be reproduced in isolation: https://github.com/eslint/typescript-eslint-parser#integration-tests

If this is still an issue for you using the latest version, please create a failing integration test for it and open a new issue to discuss.

Thanks!
